mysql_init.sls

```diff
@@ -0,0 +1,95 @@
+install_mysql:
+  pkg.installed:
+    - pkgs:
+      - mysql-server
+      - python-mysqldb 
+
+reload-mysql:
+  service.running:
+    - name: mysql
+    - enable: True
+    - reload: True
+
+root_user:
+  mysql_user.present:
+    - name: 'root'
+    - password: 'TOPSECRET'
+
+mysql remove anonymous users:
```

Comment:
I guess this is coming from the default configuration of debian package. Do we need to remove that?

Reply:
In reference to this https://mariadb.com/kb/en/library/mysql_secure_installation/ which says we can improve the security of your MariaDB installation in the following ways:
    You can set a password for root accounts.
    You can remove root accounts that are accessible from outside the local host.
    You can remove anonymous-user accounts.
    You can remove the test database, which by default can be accessed by anonymous users. 

So shall I keep it ?